examples/NavigationPlayground/js/ModalStack.js

```diff
@@ -14,32 +14,44 @@ const MyNavScreen = ({ navigation, banner }) => (
         top: navigation.state.routeName === 'HeaderTest' ? 'always' : 'never',
       }}
     >
-      <SampleText>{banner}</SampleText>
+    <SampleText>{banner}</SampleText>
```

Comment:
@JulianKingman Fix the indentation of SafeAreaView's children components

Reply:
fixed